students/13th/kimhanna/main.html

```diff
@@ -0,0 +1,205 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width" />
+    <title>repl.it</title>
+    <link href="style/common.css" rel="stylesheet" type="text/css" />
+    <link href="style/main.css" rel="stylesheet" type="text/css" />
+  </head>
+  <body>
+    <nav>
+      <div class="nav-container">
+        <div class="nav-1">
+          <img
+            class="logo_instagram"
+            src="img/logo_text.png"
+            alt="logo_instagram"
+          />
+        </div>
+        <div>
+          <input
+            id="searchArea"
+            type="search"
+            class="input-search"
+            placeholder="   검색"
```

Comment:
```suggestion
            placeholder="검색"
```
input 태그 내에서 띄어쓰기로 간격조정하지 마시고 ::placeholder 가상 선택자 사용해보세요!